### PR TITLE
PR to staging branch

### PR DESCRIPTION
**Pull request by brendajin**

### CODE REVIEW

**Comment by brendajin** [comment]
@kstack7 does that look right to you now?

**Comment by kstack7** [comment]
@brendajin hmmm seems like we're merging back production commits into staging...probably because of your production branch matching origin production? Any way you can do a fresh pull of staging and applying a new branch so we don't loop commits back into staging? More of a nit-pick for sure, but keeps the commit history cleaner. Happy to hear suggestions or counters :P :)

**Comment by brendajin** [comment]
Sorry, not used to this workflow. Let me try again.

**Comment by kstack7** [comment]
@brendajin it's all good! Your code changes would have been pretty straight-forward and safe to apply directly to production, but we're maintaining this staging > production flow for now. Usually more handy for more complex changes so I'm feeling a tad nit-picky for that reason :P I appreciate your help!

**Comment by kstack7** [comment]
Thanks again 👍
